pkgs/development/python-modules/flask/default.nix

```diff
@@ -2,12 +2,12 @@
 , itsdangerous, click, werkzeug, jinja2, pytest }:
 
 buildPythonPackage rec {
-  version = "1.0.3";
+  version = "1.1.0";
```

Comment:
Version 1.1.1 was released 3 days after your sent this PR. Might be a chance to upgrade this before it gets merged.